fed/mosaic-sumo/src/test/java/org/eclipse/mosaic/fed/sumo/util/SumoRouteFileCreatorTest.java

```diff
@@ -54,69 +56,56 @@
 
     private final static String ROUTES_FILE_XSD = "/xsd/routes_file.xsd";
 
-    @Rule
-    public TestFileRule testFileRule = new TestFileRule()
-            .with("output.rou.xml", "/rou-file-with-veh-type.rou.xml");
+    private final TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+    private final TestFileRule vehicleTypesRouteFile = new TestFileRule(temporaryFolder)
+            .with("test_vTypes.rou.xml", "/route-file-creator/test_vTypes.rou.xml");
+
+    private final TestFileRule departureFile = new TestFileRule(temporaryFolder)
+            .with("departures.rou.xml", "/route-file-creator/departures.rou.xml");
+
+    private final TestFileRule testSumoConfig = new TestFileRule(temporaryFolder)
+            .with("test.sumocfg", "/route-file-creator/test.sumocfg");
```

Comment:
The TestFileRule supports multiple files: 
```java
private final TestFileRule testFiles = new TestFileRule()
            .with("test_vTypes.rou.xml", "/route-file-creator/test_vTypes.rou.xml")
            .with("departures.rou.xml", "/route-file-creator/departures.rou.xml")
            .with("test.sumocfg", "/route-file-creator/test.sumocfg");
```

Reply:
Thanks, I don't know how I missed this 😣